tests/test_infer/test_chatglm2_infer.py

```diff
@@ -13,13 +13,12 @@
 from colossalai.testing import clear_cache_before_run, parameterize, rerun_if_address_is_in_use, spawn
 
 try:
-    import lightllm 
```

Comment:
If removing this, `HAS_LIGHTLLM_KERNEL` may be always true

Reply:
Fixed